apps/src/code-studio/pd/application_dashboard/quick_view.jsx

```diff
@@ -60,7 +50,9 @@ export default class QuickView extends React.Component {
   state = {
     loading: true,
     applications: null,
-    filter: null
+    filter: null,
+    regionalPartnerName: this.props.route.regionalPartnerName,
+    regionalPartnerValue: null
```

Comment:
It has a name but no value initially?

Reply:
Yes - it always has a name, which is displayed as the header, but it will only have a value once something is selected in the regional partner dropdown.